### ISSUE TITLE
Ctrl+= does not increase font size by default, need to set keybinding in settings.json for it to work

### ISSUE THREAD

**Issue by corneliusroemer** [microsoft/terminal#7899]
# Environment

```none
Windows build number: 10.0.18363.0 Microsoft Windows NT
Windows Terminal version (if applicable): Windows Terminal (Unpackaged) Version: 1.2.200825001-release1.2
```

# Steps to reproduce
Try to increase font size by pressing `<C>+=`

# Expected behavior
Font size increases

# Actual behavior
Font size does not increase

# Additional observations
- Font size decrease with  `<C>+-` works. So does reset with  `<C>+0`
- Manually mapping  `<C>+=` to increase font in `settings.json` works as a workaround - but it shouldn't be necessary, this should be default setting. This is what made it work: { "command": { "action": "adjustFontSize", "delta": 1 }, "keys": "ctrl+=" }

**Comment by DHowett** [comment]
> ```
> Windows Terminal version (if applicable): Windows Terminal (Unpackaged) Version: 1.2.200825001-release1.2
> ```

First, you're using a version of Terminal that's a couple months out of date (and you're using it in an unexpected/unusual configuration (Unpackaged)).

Puzzlingly, this binding _is_ in the default set. I wonder what's up.

**Comment by corneliusroemer** [comment]
TL;DR: When upgrading to v1.3 and installing by way of `.msixbundle` as opposed to rename/unzip/repeat it works by default.

I'm happy to explain what the original setup was nonetheless.

There was an AppLocker issue, so I installed by manually unzipping as you explained here: https://github.com/microsoft/terminal/issues/6046. That's why it's `Unpackaged`.

I don't know why I installed the old version, that just happened so, I guess. BUT, thanks for pointing it out, I now tried installing the new `.msixbundle` v1.3 and tada now AppLocker no longer complained and the problem is solved.
